src/utils/interpret-svg.js

```diff
@@ -303,13 +303,39 @@ var applySvgAttributes = function(node, elem, parentStyles) {
       case 'className':
         elem.classList = value.split(' ');
         break;
+      case 'x':
+      case 'y':
+        elem.translation[key] = parseFloat((value + '').replace('px', ''));
+        break;
     }
   }
 
   return styles;
 
 };
 
+/**
+ * @name Utils.updateDefsCache
+ * @function
+ * @param {SvgNode} node - The SVG Node with which to update the defs cache.
+ * @param {Object} Object - The defs cache to be updated.
+ * @description Update the cache of children of <defs /> tags.
+ */
+var updateDefsCache = function(node, defsCache) {
+  for (var i = 0, l = node.childNodes.length; i < l; i++) {
+    var n = node.childNodes[i];
+    if (!n.id) continue;
+
+    var tag = n.nodeName;
+    if (!tag) continue;
+
+    var tagName = tag.replace(/svg:/ig, '').toLowerCase();
```

Comment:
I think you can use [`localName`](https://developer.mozilla.org/en-US/docs/Web/API/Element/localName) for this

Reply:
Done